packages/desktop-client/bin/build-browser.js

```diff
@@ -0,0 +1,25 @@
+#!usr/bin/env node
+const { fsUtil, webpackUtil, shellUtil, Path } = require('../../../bin/utils');
+
+const ROOT = process.cwd();
+
+async function main() {
+  const version = await fsUtil.getVersion(Path.join(ROOT, '../package.json'));
+
+  console.log('Building version ' + version + ' for the browser...');
+
+  const env = {
+    IS_GENERIC_BROWSER: 1,
+    INLINE_RUNTIME_CHUNK: false,
+    REACT_APP_BACKEND_WORKER_HASH: await webpackUtil.getWorkerFileHash(
+      Path.join(ROOT, '/public/kcab'),
```

Comment:
I don’t want people to accidentally read this as an absolute path

```suggestion
      Path.join(ROOT, 'public/kcab'),
```